lib/collection/src/operations/shared_storage_config.rs

```diff
@@ -36,6 +38,8 @@ impl Default for SharedStorageConfig {
             update_concurrency: None,
             is_distributed: false,
             default_shard_transfer_method: None,
+            incoming_shard_transfers_limit: Some(1),
+            outgoing_shard_transfers_limit: Some(1),
```

Comment:
Based on the description in `config.yaml`, I thought default was `None`?

Also: 

```suggestion
            incoming_shard_transfers_limit: default_io_shard_transfers_limit(),
            outgoing_shard_transfers_limit: default_io_shard_transfers_limit(),
```

Reply:
The default is 1 indeed, not `None`. But a user could provide `None` to remove any limits.

I'll update the comments in `config.yaml` to clarify this better.